logexp/README.md

```diff
@@ -0,0 +1,40 @@
+# LogExp
+
+LogExp is a tool that runs post-test on our kubernetes test clusters.
+It does the job of computing the set of logfiles to be exported (based on the
+node type (master/node), cloud provider, and the node's system services),and
+then actually exports them to the GCS path provided to it.
+
+The following flags need to be supplied to the tool:
```

Comment:
I don't think we should explain flags of the tool in Readme - if you need to understand them the binary itself should expose help (and if you're using standard flag package it does).